run_yolo: false

yolo_model:
  names_file:
    name: obj.names
  cfg_file:
    name: yolov3-tiny-scale_truck.cfg
  weights_file:
    name: yolov3-tiny-scale_truck.weights

subscribers:
  prec_camera_reading:
    topic: /preceding_truck_image
    queue_size: 1
  front_camera_reading:
    topic: /usb_cam/image_raw
    queue_size: 1
  run_yolo_:
    topic: /run_yolo_flag
    queue_size: 1

publishers:
  bounding_box:
    topic: /yolo_object_detection/bounding_box
    queue_size: 1

image:
  width: 640
  height: 480

image_view:
  enable_opencv: false
  wait_key_delay: 1
  enable_console_output: false
